providers/scaleway/flags.go

```diff
@@ -0,0 +1,87 @@
+package scaleway
+
+import (
+	"os"
+
+	"github.com/scaleway/scaleway-sdk-go/scw"
+	"github.com/urfave/cli/v2"
+)
+
+const category = "Scaleway"
+
+//nolint:mnd
+var ProviderFlags = []cli.Flag{
+	&cli.StringFlag{
+		Name:    "scw-access-key",
+		Usage:   "Scaleway IAM API Token Access Key",
+		EnvVars: []string{"WOODPECKER_SCW_ACCESS_KEY"},
+		// NB(raskyld): We should recommend the usage of file-system to users
+		// Most container runtimes support mounting secrets into the fs
+		// natively.
```

Comment:
Sounds like a general approach to use secret mounts instead of envs (if I get this comment), rather than a scaleway related thing.

```suggestion
```